src/openrct2/ride/Vehicle.cpp

```diff
@@ -1945,6 +1945,21 @@ static SoundIdVolume sub_6D7AC0(
     return { currentSoundId, currentVolume };
 }
 
+SoundIdVolume Vehicle::GetLiftHillSound(Ride* curRide)
+{
+    SoundIdVolume resultSounds{};
```

Comment:
FYI, I gotta come back and change this because it may wipe out the default setting of Null and 255 set in the larger function (old lines 5509-5510).